### PR TITLE
Write HTML directly inside parser.py, eliminating convert_to_html.py

### PR DESCRIPTION
**Pull request by timhutton**
Motivation:
- eliminates a source file, making it easier to use
- user doesn't need to install `markdown` package
- we have more control over the HTML that we output

It produces a single large HTML file (as convert_to_html.py did) which might be a problem for large archives. We may need to split it up later.

### CODE REVIEW

File: parser.py [hunk 1]
```diff
@@ -120,19 +130,43 @@ def tweet_json_to_markdown(tweet, username, archive_media_folder, output_media_f
...
+    output_html_filename = 'TweetArchive.html'
+
+    HTML = """\
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet"
+          href="https://unpkg.com/@picocss/pico@latest/css/pico.min.css">
```
Comment (timhutton) [inline parser.py:160]: Can we make a simple CSS to avoid requiring an online one?

File: parser.py [hunk 2]
```diff
@@ -42,26 +42,30 @@ def extract_username(account_js_filename):
...
+    body_markdown = tweet['full_text']
+    body_html = tweet['full_text']
     tweet_id_str = tweet['id_str']
     # replace t.co URLs with their original versions
     if 'entities' in tweet and 'urls' in tweet['entities']:
         for url in tweet['entities']['urls']:
             if 'url' in url and 'expanded_url' in url:
-                body = body.replace(url['url'], url['expanded_url'])
+                body_markdown = body_markdown.replace(url['url'], url['expanded_url'])
+                body_html = body_html.replace(url['url'], url['expanded_url'])
```
Comment (timhutton) [inline parser.py:59]: Linkify URLs in the HTML

File: parser.py (hunk 1 above)
Comment (timhutton) [reply parser.py:162]: The fallback when offline (tested in flight mode) is fine, being just plain HTML.

**Review by twoscomplement — APPROVED** [review]
Looks like a good change to me!

**Pull request merged**